build.sbt

```diff
@@ -55,6 +55,7 @@ lazy val `cats-effect` = module(project) in file("modules/cats-effect")
 lazy val `cats-effect2` = module(project) in file("modules/cats-effect2")
 lazy val circe = module(project) in file("modules/circe")
 lazy val cron4s = module(project) in file("modules/cron4s")
+lazy val doobieHikari = module(project) in file("modules/doobie-hikari") dependsOn (generic)
```

Comment:
Would it make sense for this to just be pureconfig-doobie? I'm not familiar with Doobie's modules, so I'm not sure if making it general would later have us pushing unwanted dependencies to downstream users.